src/rootcommands.cpp

```diff
@@ -375,3 +379,29 @@ void RootCommands::get_attr_complete(Completion& complete) {
     else complete.none();
 }
 
+void RootCommands::set_attr_complete(Completion& complete) {
+    if (complete == 0) {
+        completeObjectPath(complete, true, \
```

Comment:
Why the backslash?

Reply:
It's gone. I can't explain where it came from :)